include/FastBVH/BVH.h

```diff
@@ -27,13 +27,15 @@ struct Node final {
   //! The number of primitives in this node.
   uint32_t primitive_count;
 
-  //! Number of elements to skip in flattened tree to get to a left child's sibling.
-  //! (Node+1 == Node's left child , Node + rightOffset == Node's right child)
-  uint32_t right_offset;
+  //! The index to the lower node.
+  uint32_t lower_node;
```

Comment:
Since the left and right indices of LBVH can be different from what's described by "right_offset", I had to make two different indices here. I plus side is that now a single node aligns to a very common cache line size of 64 bytes. I didn't measure if this actually makes a difference though, since that wasn't the reason for the change.